packages/cloud-cognitive/src/components/SidePanel/SidePanel.js

```diff
@@ -91,6 +91,24 @@ export let SidePanel = React.forwardRef(
       }
     }, [selectorPrimaryFocus, open, animationComplete]);
 
+    useEffect(() => {
+      if (open && actions && actions.length && animationComplete) {
+        const sidePanelOuter = document.querySelector(`#${blockClass}-outer`);
+        const actionsContainer =
+          sidePanelOuter &&
+          sidePanelOuter.querySelector(`.${blockClass}__actions-container`);
+        let actionsHeight = actionsContainer.offsetHeight + 16; // add additional 1rem spacing to bottom padding
+        actionsHeight = `${Math.round(actionsHeight / 16)}rem`;
+        sidePanelOuter.style.setProperty(
+          `--${blockClass}--content-bottom-padding`,
+          actionsHeight
```

Comment:
Do we also need a resize observer to recalculate this? I'm thinking that when size='max', the panel width is 75%, which will vary as the window resizes, and so buttons may reflow as they get wider or narrower and change height, causing this calculation to change.